### ISSUE TITLE
Consider `qed method` for method lints

### ISSUE THREAD

**Issue by TheMC47** [isabelle-prover/isabelle-linter#26]
For example (in `HOL.Fun`):

```isabelle
qed (insert assms, simp)
```

the lint `tactic_proofs` is not reported
